Error when no input files can be found

## Issue
When I run `node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/` but `src` folder doesn't have any `.scss` files I get an error

## Error

```
No input file was found.

npm ERR! Darwin 16.5.0
npm ERR! argv "/usr/local/Cellar/node/7.4.0/bin/node" "/usr/local/bin/npm" "run" "build-css"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code ELIFECYCLE
npm ERR! frontend@0.1.0 build-css: `node-sass-chokidar --include-path ./brosif --include-path ./node_modules brosif/ -o brosif/`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the frontend@0.1.0 build-css script 'node-sass-chokidar --include-path ./brosif --include-path ./node_modules brosif/ -o brosif/'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the frontend package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-sass-chokidar --include-path ./brosif --include-path ./node_modules brosif/ -o brosif/
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs frontend
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls frontend
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/wayman/projects/cookiecutter-drf-react/frontend/npm-debug.log
```

## Expected Behavior
I get a warning that no input files detected

## Actual Behavior
I get the warning about input files but also an NPM error

___

@TomIsion

Originally reported [here](https://github.com/facebookincubator/create-react-app/issues/2380#issuecomment-304426576)

--- a/bin/node-sass-chokidar
+++ b/bin/node-sass-chokidar
@@ -355,7 +355,7 @@ function renderDir(options, emitter) {
     if (err) {
       return emitter.emit('error', util.format('You do not have permission to access this path: %s.', err.path));
     } else if (!files.length) {
-      return emitter.emit('error', 'No input file was found.');
+      return emitter.emit('warn', 'No input files were found.');
     }
 
     forEach(files, function(subject) {

